README.md

```diff
@@ -0,0 +1,47 @@
+# Demo Rollup
+
+This is a _research-only_ demo runing the Sovereign SDK, "election" example on Celestia. None of its code is
+suitable for production use. It contains known security flaws and numerous inefficiencies.
+
+## Celestia Integration
+
+The current prototype runs against Celestia-node version `v0.7.1`. This is the version used on the `arabica` testnet
+as of Mar 18, 2023.
+
+## Warning
+
+This is a research prototype. It contains known vulnerabilities and should not be used in production under any
+circumstances.
+
+## Getting Started
+
+### Set up Celestia
+
+Set up a Celestia light node running on the Arabica testnet
+
+1. Clone the repository: `git clone https://github.com/celestiaorg/celestia-node.git`.
+1. `cd celestia-node`
+1. Checkout the code at v0.7.1: `git checkout tags/v0.7.1`
+1. Build and install the celestia binary: `make build && make go-install`
+1. Build celestia's key management tool `make cel-key`
+1. Initialize the node: `celestia light init --p2p.network arabica`
+1. Start the node with rpc on port 11111: `./celestia light start --core.ip https://limani.celestia-devops.dev --p2p.network arabica --gateway --rpc.port 11111`
```

Comment:
Should it be `celestia` instead of `./celestia`?

Reply:
Fixed!